src/Altinn.Profile.Core/Telemetry/Telemetry.ContactRegistry.cs

```diff
@@ -0,0 +1,60 @@
+using System.Diagnostics;
+
+using static Altinn.Profile.Core.Telemetry.Telemetry.ContactRegistry;
+
+namespace Altinn.Profile.Core.Telemetry;
+
+/// <summary>
+/// Telemetry for the contact register update job.
+/// </summary>
+partial class Telemetry
+{
+    private void InitContactRegisterUpdateJob(InitContext context)
+    {
+        InitMetricCounter(context, MetricNamePersonAdded, init: static m => m.Add(0));
+        InitMetricCounter(context, MetricNamePersonUpdated, init: static m => m.Add(0));
+    }
+
+    /// <summary>
+    /// Increments the counter for the number of persons created.
+    /// </summary>
+    public void PersonAdded() => _counters[MetricNamePersonAdded].Add(1);
+
+    /// <summary>
+    /// Increments the counter for the number of persons updated.
+    /// </summary>
+    public void PersonUpdated() => _counters[MetricNamePersonUpdated].Add(1);
+
+    /// <summary>
+    /// Starts a telemetry activity for the contact registry update job.
+    /// </summary>
+    /// <returns>The started activity.</returns>
+    public Activity? StartContactRegistryUpdateJob()
+    {
+        var activity = ActivitySource.StartActivity($"{ActivityPrefix}.SyncContactInformationAsync");
```

Comment:
Is the purpose of this to be able to count how many times (and possibly when) the job is run ?
Do we also want to measure how long this took? (also possible to add this at a later increment) This can be found today by the duration of the request in application insights